recipe/0064-opera-one-canvas/manifest.json

```diff
@@ -0,0 +1,606 @@
+{
+  "@context": [
+    "http://www.w3.org/ns/anno.jsonld",
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "type": "Manifest",
+  "id": "https://example.org/iiif/cookbook-recipes/0064-opera-one-canvas/manifest",
+  "label": {
+    "it": [
+      "L'elisir d'amore"
+    ],
+    "en": [
+      "The elixir of love"
+    ]
+  },
+  "metadata": [
```

Comment:
Does the `metadata` block matter for this particular recipe? I think it should be reduced to one or two at most.